SpectralOps/teller
# Adding GitHub provider

Adding Github provider.

Test: 
1. Create an GitHub token and set into your environment variable as `GITHUB_AUTH_TOKEN`
2. Add GitHub provider into `.teller` file
3. Support Put, PutMapping, Delete and DeleteMapping, see example usage in README.md file


Open Issue: #45
